web/backend/routes/proxy.ts

```diff
@@ -0,0 +1,38 @@
+import { JsonSchemaToTsProvider } from '@fastify/type-provider-json-schema-to-ts'
+import { RuntimeApiClient } from '@platformatic/control'
+import { FastifyInstance } from 'fastify'
+
+export default async function (fastify: FastifyInstance) {
+  const typedFastify = fastify.withTypeProvider<JsonSchemaToTsProvider>()
+  const api = new RuntimeApiClient()
+
+  typedFastify.all('/proxy/:pid/services/:serviceId/*', {
+    schema: {
+      hide: true, // needed since the client generation fails to properly handle the '*' wildcard
+    }
+  }, async (request, reply) => {
+    const { pid, serviceId, '*': requestUrl } = request.params as { pid: number, serviceId: string, '*': string } // cast needed because we can't define a valid json schema with the '*' wildcard
+
+    delete request.headers.connection
+    delete request.headers['content-length']
+    delete request.headers['content-encoding']
+    delete request.headers['transfer-encoding']
+
+    const injectParams = {
+      method: request.method,
+      url: '/' + requestUrl,
+      headers: request.headers,
+      query: request.query,
+      body: JSON.stringify(request.body)
```

Comment:
This is incorrect. It would only transfer JSON bodies. You'd need to add a content type parser for `*`, collect the body and pass it down.